Formula/freecad.rb

```diff
@@ -1,15 +1,13 @@
 class Freecad < Formula
   desc "Parametric 3D modeler"
   homepage "http://www.freecadweb.org"
-  version "0.19"
+  version "0.19.1"
```

Comment:
Can you update it into line 97 also ? We perhaps could use a variable approach instead by the way instead of having that value hardcoded

Reply:
yeah i think the most painless solution would be to drop the `.1` as you suggested. 👍